planetExpress/Game.java

```diff
@@ -0,0 +1,50 @@
+package planetExpress;
+import java.util.Scanner;
+public class Game {
+
+	public static void main(String[] args) 
+	{
+		PlanetExpress planet;
+		Score score;
```

Comment:
At first I wasn't sure if I like the `Score` class in the `Game`, or if it would make more sense to be in the `PlanetExpress` class. I think that after some thought this makes sense to be in the `Game` because then you could have a separate `PlanetExpress` instance that wasn't so much a game, but just a separate class.